modules/ROOT/partials/configuration/spellchecker_select_languages.adoc

```diff
@@ -4,7 +4,7 @@ IMPORTANT: This option will be removed in {productname} 6.0.
 
```

Comment:
This file needs an ID added to the option. e.g:
```
[[spellchecker_select_languages]]
=== `spellchecker_select_languages`
```

Reply:
Done